packages/manager/apps/dedicated/client/app/account/account.bundle.js

```diff
@@ -0,0 +1,92 @@
+import './user/emails/user-emails.controller';
+import './user/emails/user-emails.routing';
+import './user/emails/user-emails.service';
+import './user/infos/user-infos.controller';
+import './user/infos/user-infos.routing';
+import './user/infos/user-infos.service';
+import './user/ip/restriction/add/user-ip-restriction-add.controller';
+import './user/ip/restriction/delete/user-ip-restriction-delete.controller';
+import './user/ip/restriction/user-ip-restriction.controller';
+import './user/ip/restriction/user-ip-restriction.service';
+import './user/newAccountForm/new-account-form.config';
+import './user/password/user-password.controller';
+import './user/security/2fa/user-security-2fa.controller';
+import './user/security/2fa/user-security-2fa.routing';
+import './user/security/backupCode/delete/user-security-backupCode.controller';
+import './user/security/backupCode/manage/user-security-backupCode-manage.controller';
+import './user/security/backupCode/user-security-backupCode.controller';
+import './user/security/backupCode/user-security-backupCode.service';
+import './user/security/sms/add/user-security-sms-add.controller';
+import './user/security/sms/delete/user-security-sms-delete.controller';
+import './user/security/sms/edit/user-security-sms-edit.controller';
+import './user/security/sms/user-security-sms.controller';
+import './user/security/sms/user-security-sms.service';
+import './user/security/totp/add/user-security-totp-add.controller';
+import './user/security/totp/delete/user-security-totp-delete.controller';
+import './user/security/totp/edit/user-security-totp-edit.controller';
+import './user/security/totp/user-security-totp.controller';
+import './user/security/totp/user-security-totp.service';
+import './user/security/u2f/add/user-security-u2f-add.controller';
+import './user/security/u2f/delete/user-security-u2f-delete.controller';
+import './user/security/u2f/edit/user-security-u2f-edit.controller';
+import './user/security/u2f/user-security-u2f.controller';
+import './user/security/u2f/user-security-u2f.service';
+import './user/security/user-security.controller';
+import './user/security/user-security.routing';
+import './user/support-level/support-level.component';
+import './user/support-level/support-level.constants';
+import './user/support-level/support-level.controller';
+import './user/support-level/support-level.routing';
+import './user/support-level/SupportLevel.class';
+import './user/user.controller';
+import './user/user.routes';
+import './user/users/add/user-users-add.controller';
+import './user/users/delete/user-users-delete.controller';
+import './user/users/disable/user-users-disable.controller';
+import './user/users/enable/user-users-enable.controller';
+import './user/users/group.service';
+import './user/users/update/user-users-update.controller';
+import './user/users/users.controller';
+import './user/users/users.routing';
+import './user/users/users.service';
```

Comment:
Did you create a "moulinette" for this ? 😅

Reply:
I wish I had...